src/_cffi_src/openssl/ssl.py

```diff
@@ -252,7 +252,7 @@
                                          SSL *,
                                          const char *,
                                          unsigned char *,
-                                         int
+                                         unsigned int
```

Comment:
https://www.openssl.org/docs/man1.1.0/ssl/SSL_CTX_set_psk_server_callback.html shows this as just `int`?

Reply:
The code uses `unsigned int`, so any deviation here is just a documentation error. git says it was fixed here:
https://github.com/openssl/openssl/commit/93a048a1d8f1f6307aea0c1e7854a40b2daf236f#diff-9e87706727f317514b6408167292621f

Mismatching function pointer types, even slightly like this, is undefined behavior in C (though realistically okay). This was only building because cryptography.io doesn't treat those type errors as fatal in its build. They're just flagging warnings.